cmake/modules/configuration_files.cmake

```diff
@@ -11,6 +11,8 @@
 # The following variables will be defined when this CMake module completes:
 #
 # - CONF_FILE:              List of Kconfig fragments
+# - OVERLAY_CONFIG:         List of additional Kconfig fragments
+#                           (to be mixed in with CONF_FILE)
```

Comment:
I think there is a general understanding of how `OVERLAY_CONFIG` works.
Let's just remove this part.
```suggestion
```